node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +135,10 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            var supportedTypes = _.union(_.pluck(navigator.mimeTypes, 'type'), ['text/html', 'text/plain']);
```

Comment:
Yes navigator.mimeTypes lists the types that the browser can handle via plugins, so it will tell us if a pdf or swf, etc can be displayed by the browser.